.changeset/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.2/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    {
+      "repo": "@wethegit/react-marquee"
+    }
+  ],
```

Comment:
For this to work we need `@changesets/changelog-github`. I added as a suggestion bellow